hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGenerator.java

```diff
@@ -71,50 +77,82 @@ public void init(ProcessingEnvironment env) {
             parameterTemplate = cfg.getTemplate("parameter-template-java.ftl");
             parameterTemplateCSharp = cfg.getTemplate("parameter-template-csharp.ftl");
             messageTypeTemplate = cfg.getTemplate("messagetype-template-java.ftl");
+            messageTypeTemplateCSharp = cfg.getTemplate("messagetype-template-csharp.ftl");
         } catch (IOException e) {
-            messager.printMessage(Diagnostic.Kind.ERROR, e.getMessage());
-            throw new RuntimeException(e);
+            messager.printMessage(Diagnostic.Kind.WARNING, e.getMessage());
+//            throw new RuntimeException(e);
         }
     }
 
     @Override
     public boolean process(Set<? extends TypeElement> elements, RoundEnvironment env) {
         for (Element element : env.getElementsAnnotatedWith(GenerateParameters.class)) {
-            generate((TypeElement) element);
+            generate((TypeElement) element, Lang.JAVA);
+//            if (csharpEnabled) {
+//                generate((TypeElement) element, Lang.CSHARP);
+//            }
+//            if (cppEnabled) {
+//                generate((TypeElement) element, Lang.CPP);
+//            }
+
```

Comment:
This class will be removed in the next PR, so just ignore the uncommented code etc.